examples/third_party/qt/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+cc_library(
```

Comment:
Maybe this files and qt.bzl can be moved out of the example folder. Did not get working in a first shot.